docusaurus.config.js

```diff
@@ -21,6 +21,7 @@ module.exports = {
       logo: {
         alt: "MetaGame Logo",
         src: "img/mg-crystal.png",
+        href: "/docs/home"
```

Comment:
Having added this I am not sure it is the best idea. Any thoughts would be good. @peth-yursick @peth-yursick  @Baconaetor 🙏🏻